### PR TITLE
VIEW CHANGES IN PSQL_SCRIPT BEFORE MERGE

### PR DESCRIPTION
**Pull request by itskesin**
all table insertion, creation and triggers tested on psql
- minor bugs in insertion into frommenu 
- for the query which allows restaurant staff to view the top 5 selling items in their restaurant will probably requires user input

### CODE REVIEW

**Comment by Xdecosee** [comment]
Remove deliverybonus and then cut down basedeliveryfee to $1 or $2 default

File: kesin_local/database/psql_script.sql [hunk 1]
```diff
@@ -0,0 +1,379 @@
...
+);
+
+CREATE TABLE RestaurantStaff (
+	uid         INTEGER,
+	restaurantID INTEGER NOT NULL, 
+	PRIMARY KEY (uid),
+	FOREIGN KEY (uid, restaurantID) REFERENCES Users(uid, restaurantID) ON DELETE CASCADE
+);
+
+CREATE TABLE DeliveryRiders (
```
Comment (NyanWunPaing) [inline kesin_local/database/psql_script.sql:145]: Remove either baseDeliveryFee or deliveryBonus and keep the other as $1

File: kesin_local/database/psql_script.sql [hunk 2]
```diff
@@ -0,0 +1,379 @@
...
+	orderStatus         VARCHAR(50) DEFAULT 'Pending'     NOT NULL CHECK (orderStatus in ('Pending','Confirmed','Completed','Failed')),
+	deliveryDuration    INTEGER  DEFAULT 0				  NOT NULL,
+	timeOrderPlace      TIME DEFAULT CURRENT_TIME,
+	timeDepartToRest    TIME,
+	timeArriveRest      TIME,
+	timeDepartFromRest  TIME,
+	timeOrderDelivered  TIME,
+	PRIMARY KEY (orderID),
+	FOREIGN KEY (payOption) REFERENCES PaymentOption (payOption)
+);
```
Comment (itskesin) [inline kesin_local/database/psql_script.sql:94]: Area needed

**Comment by Xdecosee** [comment]
request to add "archive" into food:

archive boolean default false

Restaurant Staff: its for updating and deleting purpose. So when staff click delete food item, then the food entry's archive will be set to true. So that past orders can still have that particular food entry and its price. 

Customers: For browsing of food, only display food where archive is false

**Comment by Xdecosee** [comment]
Ill need food id too to handle update and delete of food item. So er diagram will change to a regular one to many relation. will explain on sat

**Pull request closed without merging**